SystemInformer/include/procprv.h

```diff
@@ -181,6 +181,7 @@ typedef struct _PH_PROCESS_ITEM
         ULONG Flags;
         struct
         {
+            ULONG IsTokenAccessible : 1;
```

Comment:
I had quickly no better solution; if i do not this, there is a difference in Elevation-Column in the process list and in the Token-Tab of the process dialog: For example Unknown vs. Elevated No (Default).

Reply:
See comments elsewhere.

Reply:
Removed